Magento2/Tests/Legacy/ClassesConfigurationUnitTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento2\Tests\Legacy;
+
+use PHP_CodeSniffer\Tests\Standards\AbstractSniffUnitTest;
+
+class ClassesConfigurationUnitTest extends AbstractSniffUnitTest
+{
+    /**
+     * @inheritdoc
+     */
+    public function getErrorList($testFile = '')
+    {
+        if ($testFile === 'ClassesConfigurationUnitTest.1.xml') {
+            return [
+                22 => 1,
+                40 => 1,
+            ];
+        }
+        if ($testFile === 'ClassesConfigurationUnitTest.2.xml') {
+            return [
+                22 => 1,
```

Comment:
In lines 22 and 42 there are no classes...

Reply:
@jcuerdo sniffer returns the position of the opening tag where the issue happens